norddrop/src/device/mod.rs

```diff
@@ -406,8 +406,8 @@ fn prepare_transfer_files(
 ) -> Result<Vec<File>> {
     let mut files = Vec::new();
 
+    #[allow(unused_variables)]
```

Comment:
why was this moved?

Reply:
Because of `i` in the for loop, it's unused on Windows and generated a warning without this change